### PR TITLE
#13373: PyTorch Tracing Sweeps - Eltwise set 3

### PR DESCRIPTION
**Pull request by mouliraj-mcw**
### Ticket
Link to Github Issue #13373 

### What's changed
Added new sweeps for tracing ops

Screenshot 2024-10-08 at 5 15 03 PM

Summary : https://docs.google.com/document/d/1AfeQ5A_hPHQRcwbtbNLb8Ti6X8V2Ks-V3YroDsYBCzQ/edit?usp=sharing

### Checklist
- [x] [All Post commit CI](https://github.com/tenstorrent/tt-metal/actions/runs/11240557397)

### CODE REVIEW

**Review by Aswinmcw — APPROVED** [review]
LGTM

File: tests/sweep_framework/sweeps/eltwise/composite/binary/maximum/maximum_pytorch2.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+# Parameters provided to the test vector generator are defined here.
+# They are defined as dict-type suites that contain the arguments to the run function as keys, and lists of possible inputs as values.
+# Each suite has a key name (in this case "suite_1" and "suite_2") which will associate the test vectors to this specific suite of inputs.
+# Developers can create their own generator functions and pass them to the parameters as inputs.
+parameters = {
+    "nightly": {
+        "input_shape": [
+            {"shape1": [1, 16, 1, 60], "shape2": []},
+            # {"shape1": [1,16,s10+1], "shape2": []},
```
Comment (umadevimcw) [inline tests/sweep_framework/sweeps/eltwise/composite/binary/maximum/maximum_pytorch2.py:31]: Can we remove the commented shape? or is it required?

File: tests/sweep_framework/sweeps/eltwise/composite/binary/minimum/minimum_pytorch2.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+# Each suite has a key name (in this case "suite_1" and "suite_2") which will associate the test vectors to this specific suite of inputs.
+# Developers can create their own generator functions and pass them to the parameters as inputs.
+parameters = {
+    "nightly": {
+        "input_shape": [
+            [1, 1],
+            [10, 10],
+            [15, 15],
+            [17, 17],
+            # [s0+1, s0+1],
```
Comment (umadevimcw) [inline tests/sweep_framework/sweeps/eltwise/composite/binary/minimum/minimum_pytorch2.py:34]: same here

File: tests/sweep_framework/sweeps/eltwise/unary/sigmoid/sigmoid_pytorch2.py [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    *,
+    device,
+) -> list:
+    data_seed = random.randint(0, 20000000)
+    torch.manual_seed(data_seed)
+
+    torch_input_tensor_a = gen_func_with_cast_tt(
+        partial(torch_random, low=-100, high=100, dtype=torch.float32), input_a_dtype
+    )(input_shape)
+    torch_output_tensor = torch.nn.functional.sigmoid(torch_input_tensor_a)
```
Comment (umadevimcw) [inline tests/sweep_framework/sweeps/eltwise/unary/sigmoid/sigmoid_pytorch2.py:114]: One generic comment for all the tests, can we make use of golden function in the place of torch functions?

**Review by umadevimcw — APPROVED** [review]

File: tests/sweep_framework/sweeps/eltwise/composite/binary/maximum/maximum_pytorch2.py (hunk 1 above)
Comment (KalaivaniMCW) [reply tests/sweep_framework/sweeps/eltwise/composite/binary/maximum/maximum_pytorch2.py:31]: we are including all the shapes from md file and commenting out the shapes that cannot be tested

File: tests/sweep_framework/sweeps/eltwise/unary/sigmoid/sigmoid_pytorch2.py (hunk 3 above)
Comment (mouliraj-mcw) [reply tests/sweep_framework/sweeps/eltwise/unary/sigmoid/sigmoid_pytorch2.py:114]: Sure, Will update it.

**Review by KalaivaniMCW — APPROVED** [review]

**Review by VirdhatchaniKN — APPROVED** [review]

File: tests/sweep_framework/sweeps/eltwise/composite/binary/maximum/maximum_pytorch2.py (hunk 1 above)
Comment (umadevimcw) [reply tests/sweep_framework/sweeps/eltwise/composite/binary/maximum/maximum_pytorch2.py:31]: okay

**Review by sjameelTT — APPROVED** [review]

**Review by jdesousa-TT — APPROVED** [review]
Pls run sweeps pipeline before merging
